libs/shared/metrics/statsd/src/lib/statsd.provider.ts

```diff
@@ -30,3 +31,14 @@ export const MockStatsDFactory: Provider<StatsD> = {
     return {} as StatsD;
   },
 };
+
+export const StatsDFactoryDirect: Provider<StatsD> = {
```

Comment:
Can we name this `StatsDProvider`, and rename `MockStatsDFactory` to `MockStatsDProvider`?